tests/test_workflow.py

```diff
@@ -534,22 +532,30 @@ def test_workflow_edit_task(
 
 
 def test_workflow_import(
-    register_user,
     invoke,
-    testdata_path: Path,
-    task_factory,
-    caplog,
+    testdata_path,
+    new_name,
 ):
 
     # create project
-    PROJECT_NAME = "project_name"
+    PROJECT_NAME = new_name()
     res_pj = invoke(f"project new {PROJECT_NAME}")
     assert res_pj.retcode == 0
     project_id = res_pj.data["id"]
 
-    task_factory(
-        user_id=register_user["id"], name="task", source="PKG_SOURCE:dummy2"
+    from fractal_server.app.db import get_sync_db
```

Comment:
Remove this, and use new rules for how to import tasks (namely, modify the test JSON files)